src/components/vm/VmInitializer.tsx

```diff
@@ -49,6 +52,27 @@ export default function VmInitializer() {
   const setVmStore = useVmStore((store) => store.set);
   const { requestAuthentication, saveCurrentUrl } = useSignInRedirect();
 
+  const init = useCallback(async () => {
+    const idos = await idOSInit();
+    // const wallet = await (await near.selector).wallet();
+    // console.log("wallet: ", wallet);
+    // try {
+    //   await idos?.auth.setNearSigner(wallet); // a WalletSelector.Wallet
```

Comment:
According to fractal doc this should be a part of initialization process but this line is throwing an error:

```
Failed to set signer:  Error: The signMessage method is not supported by MyNearWallet
```
And no matter  which wallet I choose. Got the same using hereWallet.

I think this error appears because walletSelector haven't released this [PR](https://github.com/near/wallet-selector/pull/958). It's merged but haven't released yet

Reply:
After I sent the message above I checked that wallet selector made a new [release](https://github.com/near/wallet-selector/releases/tag/v8.6.0) 👏 which means I can move forward from now